### PR TITLE
chore: 0.36.0

### PR DESCRIPTION
**Pull request by caspiano**
Some backwards-compatible updates to conform to crystal v0.36.0, as well as the inclusion of the Timecop to wipe off some seconds from the specs

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,19 +5,11 @@ services:
   - docker
 
 before_install:
-  # Add rethinkdb 2.3.6
-  - docker run --net="host" -p "29015:29015" -p "28015:28015" -d acalabs/rethinkdb:2.3.6
-
-env:
-  - TRAVIS_TEST=true
+  - docker run --net="host" -p "29015:29015" -p "28015:28015" -d rethinkdb:2.4
```
Comment (dukeraphaelng) [inline .travis.yml:8]: I don't think we have to specify network host here. I forgot what the error log was but when running locally I have to leave that flag out

File: shard.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 name: placeos-models
-version: 4.10.0
+version: 4.10.1
+crystal: ~> 0.35
```
Comment (dukeraphaelng) [inline shard.yml:3]: Shouldn't this be ~> 0.36 ?

File: .travis.yml (hunk 1 above)
Comment (dukeraphaelng) [reply .travis.yml:8]: I'm pretty sure because we are already exposing the ports, so we don't need the network host flag

Comment (caspiano) [reply .travis.yml:8]: Not necessary here, although it doesn't cause problems

**Review by GabFitzgerald — APPROVED** [review]

File: shard.yml (hunk 2 above)
Comment (caspiano) [reply shard.yml:3]: either works, it still functions on 0.35

File: .travis.yml (hunk 1 above)
Comment (caspiano) [reply .travis.yml:8]: I'm adding a github action so we can bid travis farewell

**Pull request merged**